src/Acuminator/Acuminator.Analyzers/Analyzers/DAC/DacPropertyGraphUsageAnalyzer.cs

```diff
@@ -0,0 +1,135 @@
+﻿using Acuminator.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+
+namespace Acuminator.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class DacPropertyGraphUsageAnalyzer : PXDiagnosticAnalyzer
+    {
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics =>
+            ImmutableArray.Create
+            (
+                Descriptors.PX1029_PXGraphUsageInDacProperty
+            );
+
+        internal override void AnalyzeCompilation(CompilationStartAnalysisContext compilationStartContext, PXContext pxContext)
+        {
+            compilationStartContext.RegisterSyntaxNodeAction(syntaxContext => AnalyzeGraphUsageInsideDacProperty(syntaxContext, pxContext), SyntaxKind.PropertyDeclaration);
+        }
+
+        private void AnalyzeGraphUsageInsideDacProperty(SyntaxNodeAnalysisContext syntaxContext, PXContext pxContext)
+        {    
+            PropertyDeclarationSyntax node = syntaxContext.Node as PropertyDeclarationSyntax;
+            IPropertySymbol property = syntaxContext.SemanticModel.GetDeclaredSymbol(node, syntaxContext.CancellationToken);
+
+            if (property != null && IsDacProperty(property, pxContext))
```

Comment:
@metallmonster, I'm not sure what will be return by semantic model in case of cancellation, most probably null, but I would check cancellation token just in case